### ISSUE TITLE
Error when using @@id to use multiple columns as primary key

### ISSUE THREAD

**Issue by oscarteg** [graphql-nexus/nexus-plugin-prisma#663]
I have 2 models with a third model for the many to many relationship between the 2. My models looks like this. 
```
model User {
  id           Int            @default(autoincrement()) @id
}

model Team {
  id           Int            @default(autoincrement()) @id
}

model TeamRole {
  user      User     @relation(fields: [userId], references: [id])
  userId    Int
  team      Team     @relation(fields: [teamId], references: [id])
  teamId    Int
  type      RoleType
  updatedAt DateTime @updatedAt
  createdAt DateTime @default(now())

  @@id([userId, teamId])
}
```

Prisma is able to generate the schema for this but `nexus dev` gives this error.

```
Error: Your Prisma Model TeamRole does not have an @id field. It's required for nexus-prisma to work.
at Proxy.fieldPublisher (/Users/user/nexus/node_modules/nexus-prisma/src/builder.ts:465:17)
at Object.definition (/Users/user/nexus/api/graphql/models/TeamRole.ts:7:17)
at SchemaBuilder.walkOutputType (/Users/user/nexus/node_modules/@nexus/schema/src/builder.ts:1473:9)
at SchemaBuilder.walkTypes (/Users/user/nexus/node_modules/@nexus/schema/src/builder.ts:700:16)
at SchemaBuilder.getFinalTypeMap (/Users/user/nexus/node_modules/@nexus/schema/src/builder.ts:799:10)
at Object.makeSchemaInternal (/Users/user/nexus/node_modules/@nexus/schema/src/builder.ts:1597:15)
at Object.makeSchema (/Users/user/nexus/node_modules/nexus/src/runtime/schema/schema.ts:124:55)
at Object.start (/Users/user/nexus/node_modules/nexus/src/runtime/app.ts:145:54)
```

I have fixed this by adding the id and an unique contraint on `userId` and `teamId` but I think the original should work.

**Comment by beeplin** [comment]
i got this issue too

**Comment by moderndegree** [comment]
same

**Comment by AlexGodard** [comment]
Having the same issue here.

**Comment by jamesopti** [comment]
@jasonkuhrt Any thoughts on this?

**Comment by Weakky** [comment]
Thanks for reporting this issue. It's been closed by #686 and will be available in the next release.

**Comment by Weakky** [comment]
We've just released `nexus-prisma@0.15.0-next.1`.
You can try the feature by installing `nexus-prisma@next` right now. Feel free to report any bug related to it in here. I'll re-open the issue if needed 🙏
